documentation/en/installation.md

```diff
@@ -159,6 +159,11 @@ git checkout <release version>
        export CGO_CFLAGS="-D__BLST_PORTABLE__"
        ```
 
+    3. Also for old processors, if Curio cannot start because the processor generation is too old, add the variable:\
```

Comment:
```suggestion
    3. Also for old processors, if Curio cannot start because the processor generation is too old, add the variable and rebuild the binaries:\
```